src/View/Components/Imgix.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Astrotomic\Imgix\View\Components;
+
+use Astrotomic\Imgix\ImgixManager;
+use Illuminate\View\Component;
+
+class Imgix extends Component
+{
+    protected ImgixManager $imgixManager;
+    protected string $path;
+
+    protected ?string $source;
+    protected ?string $imgWidth;
+    protected ?string $imgHeight;
```

Comment:
Width and height should be integers as the `<img>` attributes and the imgix API params are in pixels.

Reply:
Fixed